tests/test_imputation/test_drop_missing_data.py

```diff
@@ -57,3 +66,62 @@ def test_non_fitted_error(df_na):
     with pytest.raises(NotFittedError):
         imputer = DropMissingData()
         imputer.transform(df_na)
+
+
+def test_threshold(df_na):
+
```

Comment:
Hi @kevinkurek

Thank you so much for the changes and the quick turn-around. Code-wise, I think this is pretty much ready. Good work.

There is one nitpick that would really help us maintain the class looking forward:

If I understand correctly:

    If threshold =1, then all variables need to have NA to be dropped, so that is, the row needs 100% of data available
    If threshold =0.5, then 50% pf variables need to have NA to be dropped, so that is, the row needs 50% of data available

But I think the following might be inheritance from the former implementation:

    If threshold=0.75, then 75% variables need to have NA to be dropped, so that is, the row needs >=25% of data available
    If threshold=0.01, then 1% variables need to have NA to be dropped, so that is, the row needs >=99% of data available

Is my logic above correct?

If yes, could I ask you to update the comments (if necessary) from this and the next test?